docs/static/gwadoc.css

```diff
@@ -2,6 +2,24 @@ body {
     font-family: sans-serif;
 }
 
+.button {
+    background-color: #4CAF50; /* Green */
+    border: none;
+    color: white;
+    padding: 15px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+  }
+#title {
```

Comment:
Can't this just be on the `<h1>` element and ditch the need for the `#title` id?